src/vcpkg/base/files.cpp

```diff
@@ -2637,7 +2639,27 @@ namespace vcpkg
                                CopyOptions options,
                                std::error_code& ec) override
         {
-#if defined(_WIN32)
+#if defined(__MINGW32__)
```

Comment:
I would prefer to replace this with a native Win32 version rather than choosing between different std::filesystem behavior; since this is a relatively high amount of logic we're never going to test. That said, I don't to block your otherwise perfectly working change forever. If it's next Tuesday afternoon (November 16) and I don't have a proposed change for you we'll merge this.